README.md

```diff
@@ -62,6 +62,17 @@ resource "nuodbaas_database" "db" {
   name         = "db"
   dba_password = "secret"
 }
+
+# Expose nuosql arguments to connect to database
+output "nuosql_args" {
+  value = <<-EOT
+  ${nuodbaas_database.db.name}@${nuodbaas_database.db.status.sql_endpoint}:443 \
+      --user dba --password ${nuodbaas_database.db.dba_password} \
+      --connection-property trustedCertificates='${nuodbaas_database.db.status.ca_pem}' \
+      --connection-property verifyHostname=true
```

Comment:
Isn't it `true` by default? https://doc.nuodb.com/nuodb/latest/database-administration/database-access-and-security/connect-to-a-database-using-tls-and-ldap/#_driver_requirements

Reply:
Good point. I'll remove it to make the argument list simpler.